app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_posts_likes_counter
+    author.increment!(:posts_count)
```

Comment:
- I think this is a mistake 🤷🏽‍♂️, don't worry, it happens to the best of us 😄. I believe this should be `update_posts_counter` and not `update_posts_likes_counter` 😉.